src/core/Microsoft.PowerToys.Settings.UI.Lib/HotkeySettingsControlHook.cs

```diff
@@ -13,17 +13,20 @@ namespace Microsoft.PowerToys.Settings.UI.Lib
 
     public delegate bool FilterAccessibleKeyboardEvents(int key, UIntPtr extraInfo);
 
-    public class HotkeySettingsControlHook
+    public class HotkeySettingsControlHook : IDisposable
     {
         private const int WmKeyDown = 0x100;
         private const int WmKeyUp = 0x101;
         private const int WmSysKeyDown = 0x0104;
         private const int WmSysKeyUp = 0x0105;
 
+#pragma warning disable CA2213 // Disposable fields should be disposed
         private KeyboardHook _hook;
+#pragma warning restore CA2213 // Disposable fields should be disposed
```

Comment:
I think I've implemented the expected disposal pattern but fxcop is still raising the warning, even after reloading/rebuilding the project. I'm wondering if I'm missing anything or if folks are able to repro this issue (you'd have to add fxcop to the project file)